server/game/gamesteps/setup/AdaptiveDeckSelectionPrompt.js

```diff
@@ -57,7 +57,7 @@ class AdaptiveDeckSelectionPrompt extends AllPlayerPrompt {
             this.game.queueStep(new ChainBiddingPrompt(this.game, player1));
         } else {
             this.game.addMessage('Players have selected different decks. Chains will not be bid.');
-            if(player1.owner !== player1.name) {
+            if(player1.owner !== player1.player) {
```

Comment:
player1.player?

Reply:
yeah adaptive object is 
`player: string,
uuid: string,
owner: string,
deckName: string`